src/Evervault/Evervault.php

```diff
@@ -80,6 +80,18 @@ public function decrypt($data) {
         return $this->httpClient->decrypt($data);
     }
 
+    public function createClientSideDecryptToken($data, $expiry) {
```

Comment:
We should make this parameter optional

```suggestion
    public function createClientSideDecryptToken($data, $expiry = null) {
```